packages/mongo-menu/src/MongoMenu.spec.tsx

```diff
@@ -18,9 +18,17 @@ describe('packages/MongoMenu', () => {
     />,
   );
 
+  const { getByText } = renderedComponent;
```

Comment:
We're trying to move away from `getByText` to rendering the component passing a `data-testid` and using `getByTestId`. `getByText` tends to be overly brittle in most cases.

Reply:
That isn't helpful here, as we're reaching pretty far into the component and examining specific links. The usage of `getByText` was actually already in this test and I just pulled it out here instead so I'd be able to use `renderedComponent.rerender()` more easily.

Reply:
Ahh, you're right 👍